docs/Operators.md

```diff
@@ -11,7 +11,7 @@ The operators in Data-Juicer are categorized into 5 types.
 | Type                              | Number | Description                                     |
 |-----------------------------------|:------:|-------------------------------------------------|
 | [ Formatter ]( #formatter )       |   7    | Discovers, loads, and canonicalizes source data |
-| [ Mapper ]( #mapper )             |   38   | Edits and transforms samples                    |
+| [ Mapper ]( #mapper )             |   39   | Edits and transforms samples                    |
```

Comment:
This number needs to be updated after merging the latest main branch into this branch. Same in the ZH docs.